portfolio/src/main/webapp/script.js

```diff
@@ -104,9 +111,15 @@ async function grabComments() {
  * Makes call to delete data servlet which erases all comments from dataStore.
  */
 async function deleteComments() {
-  if (confirm('Are you sure you want to delete all comments')) {
-    fetch('/delete-comments', {
+  if (confirm("Are you sure you want to delete all comments")) {
+    fetch("/delete-comments", {
       method: 'POST'
     }).then(() => setTimeout(grabComments, 1000));
   }
 }
+
+async function grabBlobURL(){
```

Comment:
Function name implies that it doesn't have side effects, but it does.

Reply:
done.